src/TabView.js

```diff
@@ -189,15 +189,18 @@ export default class TabView<T: *> extends React.Component<Props<T>, State> {
             ...rest,
             panX: this.state.panX,
             offsetX: this.state.offsetX,
-            children: navigationState.routes.map(route => {
+            children: navigationState.routes.map((route, index) => {
               const scene = this._renderScene({
                 ...props,
                 route,
               });
 
               if (React.isValidElement(scene)) {
                 /* $FlowFixMe: https://github.com/facebook/flow/issues/4775 */
-                return React.cloneElement(scene, { key: route.key });
+                return React.cloneElement(scene, {
+                  key: route.key,
+                  isFocused: navigationState.index === index,
```

Comment:
Please remove this. I don't want to add this prop. It'll cause re-render every tab change and defeats the purpose of using `SceneMap` or `PureComponent`